### ISSUE TITLE
episcanpy.tl.silhouette calls sklearn.metrics.silhouette_score with wrong arguments

### ISSUE THREAD

**Issue by phobos314** [colomemaria/epiScanpy#121]
[`sklearn.metrics.silhouette_score`](https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/metrics/cluster/_unsupervised.py#L39) takes two positional arguments and `metric` as an kwarg. `episcanpy` currently [tries to pass `metric` as a positional argument](https://github.com/colomemaria/epiScanpy/blob/3eb7ef9c3a4badf5be54f1d5be1283b4e8e5da0c/episcanpy/tools/_silhouette.py#L48), which fails.

**Comment by DaneseAnna** [comment]
Thank you for reporting the error. We just fixed the parameters and updated the master branch. 

Best, 
Anna
